reconcile/prometheus_rules_tester/integration.py

```diff
@@ -247,7 +247,7 @@ def check_rules_and_tests(
 
 
 def run(
-    dry_run: bool, thread_pool_size: int, cluster_name: Optional[str] = None
+    dry_run: bool, thread_pool_size: int, cluster_name: Optional[tuple[str]] = None
```

Comment:
should we also rename the parameter from `cluster_name` to `cluster_names` since it's not a single str anymore?

Reply:
also need to rename all later usages like https://github.com/app-sre/qontract-reconcile/pull/4250/files#diff-edd71a28997929cdbab0eda4cd50dc7c10880c200a93035d621c241dc8a4b18aR238

Reply:
done